library/core/src/num/int_macros.rs

```diff
@@ -2000,5 +2000,176 @@ macro_rules! int_impl {
         pub const fn max_value() -> Self {
             Self::MAX
         }
+
+        /// Adds to the current value, returning the previous value.
+        ///
+        /// This operation wraps around on overflow.
```

Comment:
Why use wrapping arithmetic? I'd expect it to use `+=` and either panic or wrap depending on the release mode.

Reply:
I'm following the precedent set by the atomic integer types, but I'm happy to change this if desired.

Reply:
Ah ok, that's probably question for @m-ou-se then.